scss/foundation-styles/mixins.scss

```diff
@@ -88,3 +88,30 @@
     }
   }
 }
+
+@mixin homepage-search-dropdown( $border-color, $dropdown ) {
```

Comment:
comment about the name of this mixin

- Let's remove `homepage` to make the name more generic (there aren't multiple versions of "search dropdown" style so we don't need to make the mixin name so specific)
- Can we rename it to`search-input-and-dropdown`? I know the name is loooong but `search-dropdown` makes me think that this mixin is for the help type dropdown only